client/src/pages/BodyParts.js

```diff
@@ -1,9 +1,47 @@
-import { ReactComponent as BodyParts_overview } from '../assets/bodyPart_overview.svg';
+import styled from 'styled-components';
+import { ReactComponent as BodyParts_overview } from '../assets/Bodypart_overview.svg';
+import { ReactComponent as Mark } from '../assets/Mark.svg';
+import background_pages from '../assets/background_pages.svg';
+
+import { Link } from 'react-router-dom';
 
 export default function BodyParts() {
   return (
-    <div>
+    <BodypartsWrapper>
+      <h1>Wähle...</h1>
+      <p>
+        welchem Bereich deines Körpers möchtest du dich heute besonders widmen?
+      </p>
       <BodyParts_overview />
-    </div>
+      <div className="Augen">
+        <Link to="/exercisesCategory/augen">
```

Comment:
Now that I'm reading this it feels a bit odd that the first part of the url is English followed by German. 
Maybe take a note that you check all your routes in the app and have them either consistently in English or German in the end